sofa-boot-project/sofa-boot-parent/pom.xml

```diff
@@ -27,7 +27,7 @@
         <maven.source.plugin>3.0.0</maven.source.plugin>
         <maven.javadoc.plugin>2.9.1</maven.javadoc.plugin>
         <jacoco.maven.plugin>0.8.7</jacoco.maven.plugin>
-        <maven.surefire.plugin>2.22.2</maven.surefire.plugin>
+        <maven.surefire.plugin>3.0.0-M6</maven.surefire.plugin>
```

Comment:
Could we use 3.0.0 release version instead?

Reply:
Addressed. The release version was failing install attempts locally, but it seems to be sorted now.